modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -15,6 +16,16 @@ def show
 
           render json: serialized.serializable_hash
         end
+
+        def create
+          status, error = AppealsApi::FileValidator.new(params[:document]).call
+
+          if status == :ok
+            render json: { document: params[:document], uuid: params[:uuid] }
```

Comment:
Have we confirmed that params[:document] is enough for the document to hit the server/be validated?

Reply:
It runs through the validation. Given the basic nature of this endpoint, the shape of the eventual and final payload will be hashed out in more detail later I'm assuming. Either way, whatever we take in, must have a document that we can pull out and send to the validator, so that's how I wrote the spec. Am I understanding your question?

Reply:
In relation, maybe the uuid will be in the headers, but since I don't think we're entirely sure exactly what we want that to look like, this was the solution. If you have a more concrete idea of what we will ask the consumer to provide, please share!